app/pods/publications/index/controller.js

```diff
@@ -97,6 +97,22 @@ export default class PublicationsIndexController extends Controller {
     this.send('refreshModel');
   }
 
+  @action
+  resetPublicationsFilter() {
+    this.publicationFilter.reset();
+    localStorage.setItem('publicationFilter', this.publicationFilter.toString());
+    this.send('refreshModel');
+  }
+
+  get isPublicationsFiltered() {
+    return !this.publicationFilter.ministerFilterOption
+    || !this.publicationFilter.notMinisterFilterOption
+    || !this.publicationFilter.pausedFilterOption
+    || !this.publicationFilter.publishedFilterOption
+    || !this.publicationFilter.toPublishFilterOption
+    || !this.publicationFilter.withdrawnFilterOption;
+  }
```

Comment:
Kan je deze naar de `PublicationFilter` class verhuizen?  Zodat je kan doen `this.publicationFilter.hasFilter()`. Lijkt mij makkelijker te maintainen op termijn als er andere filter opties worden toegevoegd/aangepast.